src/cap.rs

```diff
@@ -9,7 +9,11 @@ use kvm_bindings::*;
 
 /// Enumeration of the extension capability list that KVM exposes.
```

Comment:
Since cargo doc will put these under "Enums" category, I would remove "Enumeration of" and I woulde leave only "The list of capabilities exposed by KVM" or something of the kind.